### ISSUE TITLE
Support having devices as image files

### ISSUE THREAD

**Issue by d4s** [go-debos/debos#20]
Ability to create OS directly on block devices.

**Comment by d4s** [comment]
@stappersg no. In #50 the image file was used.
We use debos in production and do not have this issue.
But to be honestly -- I catch #50 once during last 2 months but can't reproduce it again.

**Comment by obbardc** [comment]
does the fix for #50 close this ?

**Comment by obbardc** [comment]
I don't think this really should be a feature of debos. We should support files rather than devices. Closing. Please re-open with more context if you wish.
